src/olympia/discovery/tests/test_admin.py

```diff
@@ -1226,3 +1277,58 @@ def test_can_not_delete_without_discovery_edit_permission(self):
         assert response.status_code == 403
         assert Shelf.objects.filter(pk=item.pk).exists()
         assert item.title == 'Recommended extensions'
+
+    def test_clean_criteria_search(self):
```

Comment:
because we're explicitly testing the form here, rather than the admin view, these tests should be in `test_forms.py` instead.

Reply:
I guess you can move the responses.add in the test setup there too.